vpr/src/route/rr_graph.cpp

```diff
@@ -2765,11 +2766,12 @@ static int get_opin_direct_connecions(int x,
                             }
                         }
 
+                        int target_sub_tile = z + directs[i].sub_tile_offset;
+                        if (relative_ipin >= target_type->sub_tiles[target_sub_tile].num_phy_pins) continue;
```

Comment:
It would be good to comment why this can occur (what leads to this continue firing)?